docs/content/users/quickstart.md

```diff
@@ -70,6 +70,34 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ddev launch
     ```
 
+## CiviCRM (Standalone)
+
+[CiviCRM Standalone](https://civicrm.org/blog/ufundo/next-steps-civicrm-standalone) allows running [CiviCRM](https://civicrm.org/) without a CMS.
+
+```bash
+mkdir my-civicrm-site && cd my-civicrm-site
+ddev config --project-type=php
+ddev start
+ddev exec curl -L https://download.civicrm.org/latest/civicrm-STABLE-standalone.tar.gz > civicrm-standalone.tar.gz
+ddev exec tar --strip-components=1 -xzf civicrm-standalone.tar.gz && ddev exec rm civicrm-standalone.tar.gz
+ddev exec curl -LsS https://download.civicrm.org/cv/cv.phar --create-dirs -o bin/cv
+ddev exec chmod +x bin/cv
```

Comment:
```suggestion
ddev exec "curl -LsS https://download.civicrm.org/latest/civicrm-STABLE-standalone.tar.gz -o /tmp/civicrm-standalone.tar.gz"
ddev exec "tar --strip-components=1 -xzf /tmp/civicrm-standalone.tar.gz"
ddev exec "curl -LsS https://download.civicrm.org/cv/cv.phar --create-dirs -o bin/cv && chmod +x bin/cv"
```

Reply:
Thanks! These are great improvements, and work very well.